src/sagemaker_containers/_env.py

```diff
@@ -757,6 +779,16 @@ def output_data_dir(self):  # type: () -> str
         """
         return self._output_data_dir
 
+    @property
+    def output_intermediate_dir(self):  # type: () -> str
+        """The dir to write intermediate output artifacts for them to be synced to S3.
```

Comment:
The directory for intermediate outputs artifacts that should be be synced to S3.